business/services.go

```diff
@@ -27,6 +28,7 @@ type SvcService struct {
 }
 
 type ServiceCriteria struct {
+	AllNamespaces          bool
```

Comment:
Is this being used in the PR ?
Perhaps you wanted to use it in the business/istio_validations.go ? Or that would be a followup task ?

Reply:
You are right, this is not used in this PR scope, so removing it now, will be added in further PRs if necessary.